### PR TITLE
[Card#38]-Remove invite from local db and remote api

### PR DESCRIPTION
**Pull request by HarshwardhanSingh**
Trello card: https://trello.com/c/kZhLZqMh

Branched out from **master**

Rebased with **list_calendar_members** branch

### CODE REVIEW

File: app/controllers/calendars_controller.rb [hunk 1]
```diff
@@ -94,7 +94,10 @@ def invite
 
   # Remove user from a calendar
   def remove
-    response = @client.remove_user(params[:calendar_id], params[:acl_id])
+    calendar = Calendar.find_by(google_id: decode_id(params[:calendar_id]))
```
Comment (HarshwardhanSingh) [inline app/controllers/calendars_controller.rb]: user `find_by!` instead of `find`

File: app/controllers/calendars_controller.rb [hunk 2]
```diff
@@ -94,7 +94,10 @@ def invite
...
+    invites = Invite.where(google_id: decode_id(params[:acl_id]), calendar_id: calendar.id)
```
Comment (HarshwardhanSingh) [inline app/controllers/calendars_controller.rb]: use associations to remove clear this line a bit.

File: app/controllers/calendars_controller.rb [hunk 3]
```diff
@@ -94,7 +94,10 @@ def invite
...
+    @client.remove_user(params[:calendar_id], params[:acl_id])
+    invites.each {|invite| invite.destroy}
```
Comment (deepak) [inline app/controllers/calendars_controller.rb]: can use `destroy_all` on association

File: app/services/google_calendar_service.rb [hunk 4]
```diff
@@ -54,6 +54,14 @@ def update_calendar(calendar_id, calendar)
       headers: {"Content-Type": "application/json"})
   end
 
+  def update_calendar(calendar_id, calendar)
+    calendarId = Base64.urlsafe_decode64(calendar_id)
```
Comment (deepak) [inline app/services/google_calendar_service.rb]: use snake_case for ruby variables

File: app/views/calendars/settings.html.erb [hunk 5]
```diff
@@ -21,7 +21,7 @@
           <td><%= image_tag(invite.user.avatar, class: 'img-thumb img-responsive') %></td>
           <td><%= invite.user.name %></td>
           <td>
-            <%= link_to fa_icon("times", text: "Remove"), remove_invitation_path(calendar_id: params[:calendar_id], acl_id: encode_id(invite.google_id)) %>
+            <%= link_to fa_icon("times", text: "Remove"), remove_invitation_path(calendar_id: params[:calendar_id], acl_id: encode_id(invite.google_id)) unless invite.user.id == calendar.owner_id %>
```
Comment (deepak) [inline app/views/calendars/settings.html.erb]: very minor nitpick. feel free to ignore.
can use invite.user_id here unless you want to check that the association exists

**Pull request merged**